src/main/java/org/candlepin/resteasy/filter/SecurityHoleAuthorizationFilter.java

```diff
@@ -116,10 +116,12 @@ private void validateAnonymousCloudConsumer(AnonymousCloudConsumerPrincipal prin
         AnonymousCloudConsumer anonymousCloudConsumer = principal.getAnonymousCloudConsumer();
         String ownerKey = checkCloudAccountOrganization(anonymousCloudConsumer);
         if (!ownerCurator.existsByKey(ownerKey)) {
-            throw new TooManyRequestsException(ORG_NOT_CREATED_IN_CANDLEPIN);
+            throw new TooManyRequestsException(i18nProvider.get().tr("Anonymous owner does not yet exist"))
+                .setRetryAfterTime(ORG_NOT_CREATED_IN_CANDLEPIN);
```

Comment:
This specific case can happen with non-anonymous orgs as well (in the case a proper org exists, but hasn't been created in Candlepin yet).

Reply:
Removed "Anonymous" from the message